src/pages/_app.jsx

```diff
@@ -78,12 +76,6 @@ const WrappedApp = ({ Component, pageProps }) => {
       setAmplifyConfigured(true);
     }
   }, [amplifyConfig]);
-  useEffect(() => {
-    // clear the store only if we navigate to a new experiment from data-management
-    if (router.route === '/data-management') {
-      dispatch(switchExperiment());
```

Comment:
Will the store reset when you switch an experiment?

Reply:
Yes, the call to `switchExperiment` is moved into the `AppRouteProvider` component. This action will be dispatched if the user is moving from `data-management` to other parts of the application.